rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
```

Comment:
- [ ] Add `book` and `person` to the attribute accessor as you need to getter and setter both.